SQL/0000-00-00-schema.sql

```diff
@@ -2286,6 +2287,16 @@ CREATE TABLE subproject (
 INSERT INTO subproject (SubprojectID, title, useEDC, WindowDifference) VALUES (1, 'Control', false, 'optimal');
 INSERT INTO subproject (SubprojectID, title, useEDC, WindowDifference) VALUES (2, 'Experimental', false, 'optimal');
 
+DROP TABLE IF EXISTS `Project`;
+CREATE TABLE `Project` (
+    `ProjectID` INT(2) Default NULL,
+    `Name` VARCHAR(255) NULL,
+    `recruitmentTarget` INT(6) Default NULL,
+    PRIMARY KEY (`ProjectID`)
+)ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+INSERT INTO Project (ProjectID, Name, recruitmentTarget) VALUES (1,'Sample Project' , 50);
+INSERT INTO Project (ProjectID, Name, recruitmentTarget) VALUES (2,'Another Sample Project' , 50);
```

Comment:
If I don't have those it fails the unit test which tests getprojectlist.
The test test wants to see those 2 projects.
On 18 Sep 2015 05:32, "Tara Campbell" <notifications@github.com> wrote:

> In SQL/0000-00-00-schema.sql
> <https://github.com/aces/Loris/pull/1250#discussion_r39838377>:
>
> > @@ -2286,6 +2287,16 @@ CREATE TABLE subproject (
> >  INSERT INTO subproject (SubprojectID, title, useEDC, WindowDifference) VALUES (1, 'Control', false, 'optimal');
> >  INSERT INTO subproject (SubprojectID, title, useEDC, WindowDifference) VALUES (2, 'Experimental', false, 'optimal');
> >
> > +DROP TABLE IF EXISTS `Project`;
> > +CREATE TABLE `Project` (
> > +    `ProjectID` INT(2) Default NULL,
> > +    `Name` VARCHAR(255) NULL,
> > +    `recruitmentTarget` INT(6) Default NULL,
> > +    PRIMARY KEY (`ProjectID`)
> > +)ENGINE = InnoDB  DEFAULT CHARSET=utf8;
> > +INSERT INTO Project (ProjectID, Name, recruitmentTarget) VALUES (1,'Sample Project' , 50);
> > +INSERT INTO Project (ProjectID, Name, recruitmentTarget) VALUES (2,'Another Sample Project' , 50);
>
> Do we want to have default projects inserted by the schema?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aces/Loris/pull/1250/files#r39838377>.
>